src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -197,9 +198,9 @@ protected void reloadConfig() {
     }
     try {
       log.info(String.format("Reloading hdfs-site.xml from %s", configUri));
-      String cfgCmd[] = new String[]{"sh", "-c",
-          String.format("curl -o hdfs-site.xml %s ; cp hdfs-site.xml etc/hadoop/", configUri)};
-      Process process = Runtime.getRuntime().exec(cfgCmd);
+      ProcessBuilder processBuilder = new ProcessBuilder("sh", "-c",
+          String.format("curl -o hdfs-site.xml %s ; cp hdfs-site.xml etc/hadoop/", configUri));
+      Process process = processBuilder.start();
       //TODO(nicgrayson) check if the config has changed
       redirectProcess(process);
       int exitCode = process.waitFor();
```

Comment:
What if this failed? Seems wrong to just log.info a non-zero exitCode and ignore it. Couldn't this result in a stale/missing hdfs-site.xml?